Zero-Gravity-Store-React/src/Contexts/Cart-context.js

```diff
@@ -29,8 +29,27 @@ const CartProvider = ({children})=>{
               { headers: { authorization: authState.token } }
             );
             cartDispatch({ type: "ADD_TO_CART", payload: item });
+            toast.success(' Add to Cart Successfully!!', {
+              position: "bottom-right",
+              autoClose: 5000,
+              hideProgressBar: false,
+              closeOnClick: true,
+              pauseOnHover: true,
+              draggable: true,
+              progress: undefined,
+              theme: "dark"
+              });
```

Comment:
Instead of repeating toast in all places. Make it as common function and you can use them wherever needed.

Reply:
Yeah sure I will be adding my own toast component and will make separate function then.